### DISCUSSION TITLE
Allow second row for toolbar, dont use "Show more items" button

### DISCUSSION THREAD

**Discussion by foxtrotcz** [craftcms/ckeditor#124] [Ideas] [answered]
Hi,
when there is a lot of buttons in toolbar it's user unfriendly to always have to click the button to see everything.
![image](https://github.com/craftcms/ckeditor/assets/10333810/94dc99df-3cb9-4829-98cf-c315322c1926)

It would be more comfortable if you could just use second row for the remaining buttons.

I tried to set it in config with `shouldNotGroupWhenFull: true` as shown here https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editorconfig-EditorConfig.html#member-toolbar but looks like extended config format doesnt work. 

Thanks.

**Comment by brandonkelly** [comment, marked as answer]
Made this possible for the next release! (d69afc4e9352c57d6e894cdb3a5fc3ab00c9f4cf)

**Reply by brandonkelly** [reply]
CKEditor 3.5.0 is out with this :tada:

Note that the update requires [Craft 4.5.0](https://craftcms.com/blog/craft-4-5-released).
